### PR TITLE
rebalance: average forward fee report is optional

### PR DESCRIPTION
**Pull request by gallizoltan**
- listforwards call takes half a minute on my machine and consumes lots of memory, as I have hundreds of thousands of forwards right now
- it's an option to turn this part of the report off, although by default it's turned on for backward compatibility (others may use this already)

### CODE REVIEW

**Review by m-schmoock — APPROVED** [review]
Jup, nothing wrong with that

**Comment by chrisguida** [comment]
Hey @m-schmoock can you hold off on merging more stuff for now until #484 is merged? We'd like to be able to verify that new PRs are passing CI.

**Comment by chrisguida** [comment]
@m-schmoock no worries!

If you have a moment, I'd love a review on the #484 :)

**Pull request merged**
